ghost/admin/app/components/members/list-item.hbs

```diff
@@ -29,15 +29,13 @@
         </LinkTo>
     {{/if}}
     {{#if @newsletterEnabled}}
-        {{#if (feature "emailAnalytics")}}
-            <LinkTo @route="member" @model={{@member}} class="gh-list-data middarkgrey f8 {{unless @member.name "gh-members-list-open-rate-noname"}}" data-test-table-data="open-rate">
-                {{#if (not (is-empty @member.emailOpenRate))}}
-                    <span>{{@member.emailOpenRate}}%</span>
-                {{else}}
-                    <span class="midlightgrey">N/A</span>
-                {{/if}}
-            </LinkTo>
-        {{/if}}
+        <LinkTo @route="member" @model={{@member}} class="gh-list-data middarkgrey f8 {{unless @member.name "gh-members-list-open-rate-noname"}}" data-test-table-data="open-rate">
+            {{#if (not (is-empty @member.emailOpenRate))}}
+                <span>{{@member.emailOpenRate}}%</span>
+            {{else}}
+                <span class="midlightgrey">N/A</span>
+            {{/if}}
+        </LinkTo>
```

Comment:
Is the removal of the feature flag here intentional? `emailAnalytics` was a special-case feature because it wasn't a labs flag but a user-selectable option. The original idea was that all email analytics UI is hidden when email analytics is disabled but maybe that's changed now? I'm not seeing `emailAnalytics` really used anywhere else so it might just be we need another cleanup task for it.

Reply:
Hmmm, the `Open rate` header in the table doesn't have this check. And I was seeing this column locally so that got me confused I think. I had the impression `feature "emailAnalytics"` always returned true. Gonna check what is happening here, brb

Reply:
So the emailAnalytics feature isn't anymore, but it now maps to config.emailAnalytics, which is true by default:
https://github.com/SimonBackx/Ghost/blob/641c87c5d744997788e61b2b5573b4aae7ec68b3/ghost/core/core/shared/config/defaults.json#L158

I'm gonna create a cleanup issue to replace it with`settings.emailTrackOpens` || `settings.emailTrackClicks` + check if we still need it in the backend. I think the config is still used for enabling/disabling the background job

Reply:
I think on this place it is good to remove it, because otherwise it wouldn't add the column in the rows when it did add the column header in the table.